rust/rsc/.config.json

```diff
@@ -1,6 +1,6 @@
 {
   "database_url": "postgres://localhost:5433/test",
-  "server_addr": "127.0.0.1:3001",
+  "server_addr": "0.0.0.0:3002",
```

Comment:
Is this just an unrelated change sneaking in?

Reply:
Yeah, this is just the default config that I keep stashing/unstashing for my local testing. It made it in by accident and I figured I should just keep it since its the default testing values